### PR TITLE
added write_empty_chunks to ZARR_OPTIONS

### LINKED ISSUE #94: Skip writing fill-value-only chunks
It would be really nice if `rechunker` could exploit the Zarr feature that chunks comprising only fill-values may be missing entirely, given that a `fill_value` is defined for that array. With a new configuration parameter, e.g. `drop_fill_value_chunks`,  `rechunker` could be configured not to write a target chunk at all, given all of its values equal `fill_value`. 

In our `xcube` package we have a dedicated tool called [xcube prune](https://xcube.readthedocs.io/en/latest/cli/xcube_prune.html) for this purpose. But it may be slow for large datasets because it tests and deletes chunks sequentially.

### PR DESCRIPTION
**Pull request by norlandrhagen**
Adds `write_empty_chunks` to ZARR_OPTIONS. This should allow the `write_empty_chunks` to be passed through to zarr. 

Similar functionality to this xarray PR: https://github.com/pydata/xarray/pull/6348

Closes #94

### CODE REVIEW

**Comment by norlandrhagen** [comment]
cc @andersy005

**Comment by andersy005** [comment]
The `write_emtpy_chunks` option is available starting with `xarray>2022.3` and [`zarr>=2.11`](https://zarr.readthedocs.io/en/stable/release.html#release-2-11-0). However, rechunker doesn't enforce lower bounds on `zarr` and `xarray` versions. should we add a check for this? cc @rabernat

**Comment by rabernat** [comment]
Thanks for this @norlandrhagen!

It's fine with me if we depend on a minimum zarr version.

**Comment by norlandrhagen** [comment]
Added minimum versions for dask and zarr in setup.py. LMK if this is the wrong place!

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,12 @@
     long_description = f.read()
 
 
-install_requires = ["dask[array,diagnostics]", "zarr", "xarray", "mypy_extensions"]
+install_requires = [
+    "dask[array,diagnostics]",
+    "zarr>=2.11",
+    "xarray>2022.3",
```
Comment (rabernat) [inline setup.py:13]:
```suggestion
    "xarray>=2022.3",
```

**Comment by rabernat** [comment]
Sorry for letting this hang for so long! 🤦 

If we add these minimum versions, it looks like we also have to drop python 3.7 support. That's fine with me.

**Comment by rabernat** [comment]
I will merge this and then make a new PR for dropping python 3.7.

**Pull request merged**